src/contracts/MarketCalculations.ts

```diff
@@ -44,14 +44,14 @@ export const closePosition = (
   slippage: number,
 ): ClosePositionReturn => {
   const aToSwap = optimalSwap(aPool, bPool, aHoldings);
-  const aToSwapWithSlippage = aToSwap + (aToSwap * slippage) / 100;
-  const aLeft = aHoldings - aToSwapWithSlippage;
-  const bReceived = fixedInSwap(aPool, bPool, aToSwapWithSlippage);
+  const bReceived = fixedInSwap(aPool, bPool, aToSwap);
+  const bReceivedWithSlippage = bReceived - (bReceived * slippage) / 100;
```

Comment:
@shekhar-shubhendu , the formula should still be `bReceived - (bReceived * slippage) / 100` or it should be changed to `bReceived + (bReceived * slippage) / 100`

Reply:
`minus` since we are accepting `min_output`